catalog/static/post-message.html

```diff
@@ -0,0 +1,8 @@
+<!doctype html>
+<html lang="en">
+  <body>
+    <script>
+      window.opener.postMessage(window.location.hash, window.location.origin);
```

Comment:
if using postMessage, i think we should use some `type` field to help the consuming code filter messages and ensure it's the proper one

Reply:
Ok